packages/core/src/indexing/database-index-api.ts

```diff
@@ -5,6 +5,7 @@ export interface IndexStreamArgs {
   readonly streamID: StreamID
   readonly model: StreamID
   readonly controller: string
+  readonly streamContent: string
```

Comment:
```suggestion
  readonly streamContent: Record<string, any>
```

Reply:
I think the content should probably get passed down as an object as far down the stack as possible, and then individual database API implementations can decide how to translate the content into a format the database understands.  If we had a MongoDB database backend for example, you wouldn't want to send it a string, the mongodb Node.js client has support for receiving a javascript object directly.